### PR TITLE
Upgrade packages

### PR DESCRIPTION
**Pull request by a2441918**
# Pull Request Template

The code review checklist below is used for all pull requests.

 1. Review the list before submitting your pull request.
 2. Leave the list intact for the code reviewer's use.

## Checklist

- [x] Latest code from master has been merged into the pull request branch
- [x] Honors [the seven code virtues](https://pragprog.com/magazines/2011-08/how-virtuous-is-your-code)
  - [ ] Working, as opposed to incomplete
  - [ ] Unique, as opposed to duplicated
  - [ ] Simple, as opposed to complicated
  - [ ] Clear, as opposed to puzzling
  - [ ] Easy, as opposed to difficult
  - [ ] Developed, as opposed to primitive
  - [ ] Brief, as opposed to chatty
- [x] Code is camelCased
- [x] No commented out code (if required, place // TODO above with explanation)
- [x] No linting issues
- [x] Automated tests exist and pass
- [x] Build is successful (`npm run build`)
- [x] Works in IE 11, Chrome, Firefox, and Edge

## Thanks!

:heart:

### CODE REVIEW

File: webpack.config.prod.js [hunk 1]
```diff
@@ -148,7 +148,9 @@ export default {
           }, {
             loader: 'sass-loader',
             options: {
-              includePaths: [path.resolve(__dirname, 'src')],
+              sassOptions: {
+              includePaths: [path.resolve(__dirname, 'src')]
```
Comment (kwelch) [inline webpack.config.prod.js]: fix indenting please.

**Comment by kwelch** [comment]
Note the `package.json` had an update when I ran it locally. I am never sure is that is expected.

**Comment by a2441918** [comment]
@kwelch I think its because of a new commit I pushed

**Comment by a2441918** [comment]
Any updates on this PR?

**Comment by kwelch** [comment]
I think we need @coryhouse since appveyor did not report back properly and I am unable to merge this.

**Comment by nickytonline** [comment]
Yeah, I wonder why AppVeyor always hangs. 🤔

**Comment by coryhouse** [comment]
Works great for me! Thanks for your help!

I'm going to merge as is - Appveyor has been failing for awhile for unrelated reasons that I'll dig into separately.

**Comment by nickytonline** [comment]
Thanks Cory!

**Pull request merged**